waggle-dance-core/src/main/java/com/hotels/bdp/waggledance/mapping/service/GrammarUtils.java

```diff
@@ -97,19 +97,20 @@ public static Map<String, String> selectMatchingPrefixes(Set<String> prefixes, S
       return matchingPrefixes;
     }
 
-    Map<String, List<String>> prefixPatterns = new HashMap<>();
+    Map<String, Set<String>> prefixPatterns = new HashMap<>();
     for (String subPattern : OR_SPLITTER.split(dbPatterns)) {
       for (String prefix : prefixes) {
-        String[] subPatternParts = splitPattern(prefix, subPattern);
```

Comment:
is `splitPattern` still used anywhere now, maybe it can go?

Reply:
only used for UT.

Reply:
then please delete it and associated unit tests

Reply:
done.